airbyte-integrations/connectors/destination-snowflake/build.gradle

```diff
@@ -4,9 +4,9 @@ plugins {
 }
 
 airbyteJavaConnector {
-    cdkVersionRequired = '0.9.0'
+    cdkVersionRequired = '0.10.0'
     features = ['db-destinations', 's3-destinations']
-    useLocalCdk = false
+    useLocalCdk = true
```

Comment:
Don't forget to revert!

Reply:
I have a whole pile of commits with formatting/metadata/build.gradle stuff locally 😅 just waiting on the current CI run to finish before pushing to avoid cancelling+restarting the integration tests
![image](https://github.com/airbytehq/airbyte/assets/5741425/4c5c63ee-f43f-4738-a16b-ac715ac59ebb)

(also... discussion on the [cdk publish flow doc](https://docs.google.com/document/d/1NYCbp1uShNzxXFogi5W2IdHyiEqiU6LmTPPQoVsl3TQ/edit) is mostly settled :D which would solve this once and for all)